Add recursive option

gsoyka opened the pull request:
This adds the option to specify a `--recursive` flag.  If used, the parameters found will be exported as environment variables with the `/` character substituted with `_`.  Behavior without the recursive flag is unchanged.

Example:
`/parameter/key1` -> `parameter_key1`

gsoyka commented:
Not a problem, I've updated to remove the built binaries and added documentation of the feature into the readme.

orfin commented:
Thanks, Readme looks cool now but I there's a commit that reverts back that binaries? :< "Add original bin files back in"

gsoyka commented:
Yes, the first commit I removed the binaries from tracking by mistake, and the PR showed them as removed, so to fix it I pulled the original binaries from master and committed them back in so that the diff only showed changes to the two files I modified.

orfin commented:
Thanks, that's very good contribution. I've just merged it and built new binaries version. 👌
